.github/workflows/pull-request.yml

```diff
@@ -37,4 +37,17 @@ jobs:
                     packageUser: ${{ github.actor }}
                     packagePAT: ${{ secrets.GITHUB_TOKEN }}
                 run: ./gradlew clean :ballerina:testExamples --stacktrace --scan --console=plain --no-daemon --continue
-
+    windows-build-without-tests:
+        runs-on: windows-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Set up JDK 11
+              uses: actions/setup-java@v1
+              with:
+                  java-version: 11
+            - name: Build Ballerina Distribution
+              env:
+                  packageUser: ${{ github.actor }}
+                  packagePAT: ${{ secrets.GITHUB_TOKEN }}
+              run: ./gradlew.bat clean build --stacktrace --scan --console=plain --no-daemon --continue -x test
```

Comment:
Have included the windows build without tests for PRs builds as well. Need enable tests in windows in all the workflows as a separate effort 
https://github.com/ballerina-platform/ballerina-distribution/issues/1319